corehq/apps/userreports/es/columns.py

```diff
@@ -0,0 +1,26 @@
+from corehq.apps.reports.datatables import (
+    DataTablesColumn,
+    DataTablesHeader,
+    DTSortType,
+)
+
+
+class EsColumnConfig(object):
```

Comment:
does this need to be different from `SqlColumnConfig`? I guess the `.columns` are different types?